src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -306,21 +369,29 @@ export class AudioCue {
 		name: localize('audioCues.lineHasError.name', 'Error on Line'),
 		sound: Sound.error,
 		settingsKey: 'audioCues.lineHasError',
+		alertSettingsKey: AccessibilityAlertSettingId.Error,
```

Comment:
Can we combine the alertSettingsKey and the settingsKey in some future refactoring?
Maybe sth like this:
```json
"signals.lineHasError": {
   "audioCue": "on",
   "alert": "on",
}
```

Reply:
https://github.com/microsoft/vscode/issues/202018